dotnet/src/Connectors/Connectors.AzureOpenAI/CompatibilitySuppressions.xml

```diff
@@ -1,27 +1,363 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <!-- https://learn.microsoft.com/en-us/dotnet/fundamentals/package-validation/diagnostic-ids -->
 <Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
+    <DiagnosticId>CP0002</DiagnosticId>
+    <Target>M:Microsoft.SemanticKernel.AzureOpenAIKernelBuilderExtensions.AddAzureOpenAIAudioToText(Microsoft.SemanticKernel.IKernelBuilder,System.String,System.String,Azure.Core.TokenCredential,System.String,System.String,System.Net.Http.HttpClient)</Target>
```

Comment:
Given the new parameter is last and optional I don't think any code get's impacted e.g. I don't see any sample updates. So I think these compat issues are of no concern, is that correct?

Reply:
Yes, no impact in the code, made sure this change wasn't impactful, but suppression is triggered due touching the APIs signature.